zap/src/main/java/org/parosproxy/paros/control/Control.java

```diff
@@ -666,4 +668,43 @@ public void setMode(Mode mode) {
             model.getOptionsParam().getViewParam().setMode(mode.name());
         }
     }
+
+    public static LinkedList<String> getRecentSessionList() {
+        LinkedList<String> recentSessionList = new LinkedList<>();
+        for (String path :
+                Model.getSingleton()
+                        .getOptionsParam()
+                        .getConfig()
+                        .getStringArray("recentsessions.path")) {
+            recentSessionList.add(path);
+        }
+        return recentSessionList;
+    }
+
+    private void setRecentSessionList(String path) {
+        LinkedList<String> recentSessionList = getRecentSessionList();
```

Comment:
Is there a reason to do an intermediate store here and not just use `getRecentSessionList()` for subsequent actions directly?

Reply:
I changed it to static, works fine :)